src/jdmtool/skybound.py

```diff
@@ -7,6 +8,15 @@
 class SkyboundException(Exception):
     pass
 
+@dataclass
+class MemoryCard:
+    type: str
+    iid: int
+    memory_layout: list[int]
```

Comment:
`list[int]` requires Python 3.9 (I think); I've been keeping jdmtool compatible with 3.7.

Reply:
you might consider some guidelines for contributors for a bit more clarity.

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

Reply:
Sounds good! 

You're the first contributor here, so this has never come up before 🙂

Reply:
yea, i'm all good if you want to rip code solo.  I love your idea.  You have made a really cool thing.  Thank you for building it.  i'd be interested in collaborating, but don't want to waste either of our times with me writing up the wrong stuff.  I'll keep things lean.  also happy to be a real collaborator if you have any interest in that.

Reply:
Sorry, didn't mean to offend or annoy you. I appreciate your changes, and I'm definitely interested in collaborating.

I realize I can be overly picky. I also don't have much experience working on open source projects, so not really familiar with the etiquette. (E.g., I could rewrite your code to my liking instead of asking you to update it - but I don't know if that's considered acceptable? Or just make changes myself - but then I'm basically taking credit for your fix...)

Anyways, please don't take it personally. I didn't mean to waste your time.

Reply:
I would offer that you are reviewing code like I work for you, not like i'm a fellow enthusiast trying to add something useful. I can see you have an architectural theme, and work in flight.  I probably overstepped a bit.   But your manner of review, though fine professionally, doesn't really invite collaboration.  For better or worse, i'll contribute the little that i need to make your tool work for my needs and then move on.  Its is fine to disagree with my thoughts presented as code in pull requests - also fine to refactor.  Again, i'd be happy to offer "big picture" thoughts and contribute code - and even try to adhere to your style and specifics.  probably just stylistic difference. no harm no foul. As noted, I'm impressed by your tool, and appreciate you for writing it.

Reply:
That's perfectly fine.

I know you're just trying to help, and I obviously don't expect anything from you. It's just that the code is fragile as it is (given Python's lack of type safety, etc.), and I haven't even had a chance to write unit tests - so any change has a chance of breaking things - even something as trivial as `list[str]`. I have branches of my own that have been sitting out there for weeks or months because I'm not happy with the design.

And you're right, my main experience with code reviews is in professional settings; sorry if I sounded uninviting. I really didn't mean to get into an argument over something as silly as a code review.